runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -166,11 +166,11 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 		}
 #if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
 		if (try_scan(&scan_start, "enableDoubleMapping")) {
-			extensions->indexableObjectModel.setEnableDoubleMapping(true);
+			extensions->isDoubleMapRequested = true;
```

Comment:
let's be more specific and rename the control flag and the options with appending Arraylet(Double...)

status flag name is ok as is, since it's in the domain of IndexibleModel class.